engine/access/rest/routes/router.go

```diff
@@ -231,7 +236,11 @@ func normalizeURL(url string) (string, error) {
 		// address based resource. e.g. /v1/accounts/1234567890abcdef
 		parts = append(parts, "{address}")
 		if matches[0][5] == "keys" {
-			parts = append(parts, "keys", "{index}")
+			if len(matches[0][5]) != 0 {
```

Comment:
this will always be true since we already check that it `== "keys"`

Try adding some tests to cover the new endpoint here:
https://github.com/onflow/flow-go/blob/9d73bfbe266403ceeefc52ec14413e37ceb80155/engine/access/rest/routes/router_test.go#L11

and verifying it's working as expected